pkg/controller/garbagecollector/graph.go

```diff
@@ -98,32 +98,32 @@ func (n *node) isDeletingDependents() bool {
 	return n.deletingDependents
 }
 
-func (ownerNode *node) addDependent(dependent *node) {
```

Comment:
Why is the name change needed?

Reply:
maybe rename to owner, see if that satisfies the linter?

Reply:
golint suggest that reciver name should be consistent with previous receiver.
The reciver name of isDeletingDependents on line 95 is n, so we should use the same name.